### PR TITLE
Test

### PR DESCRIPTION
**Pull request by SharpSeeEr**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
 <script setup lang="ts">
 // This starter template is using Vue 3 <script setup> SFCs
 // Check out https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup
-import AppHeader from './components/AppHeader.vue'
-import CarSettingsForm from './components/CarSettingsForm.vue';
+import { reactive, ref } from 'vue';
+import AppHeader from './components/AppHeader.vue';
+import NumberInput from './components/NumberInput.vue';
+import TuneSettingsForm from './components/CarSettingsForm.vue';
```
Comment (SharpSeeEr) [inline src/App.vue:15]: Why the rename here?

File: src/components/BuildSettingsForm.vue [hunk 2]
```diff
@@ -130,19 +130,6 @@ const {
...
-        v-model="form.build.tiresAndRims.trackWidth.front"
-        label="Front"
-        :type="TrackWidthType"
-      />
-      <EnumSelect
-        v-model="form.build.tiresAndRims.trackWidth.rear"
-        label="Rear"
-        :type="TrackWidthType"
-      />
-    </div>
```
Comment (SharpSeeEr) [inline src/components/BuildSettingsForm.vue:145]: Why was Track Width removed?

File: src/lib/generator.ts [hunk 3]
```diff
@@ -212,17 +213,48 @@ function formatConversions(build: BuildSettings): string[] {
...
+      ['Displacement', build.engine.displacement],
+      ['Pistons', build.engine.pistons],
+      ['Turbo', build.engine.turbo],
+      ['Twin Turbo', build.engine.twinTurbo],
+      ['Supercharger', build.engine.supercharger],
+      ['Centrifigul Supercharger', build.engine.centrifigulSupercharger],
+      ['Intercooler', build.engine.intercooler],
+      ['Oil Cooling', build.engine.oilCooling],
+      ['Flywheel', build.engine.flywheel],
+    ].filter((v) => v[1] !== Upgrade.na)),
```
Comment (SharpSeeEr) [inline src/lib/generator.ts:235]: This was all done automatically with the `formatBuildSection` function.  Any particular reason for the change?

File: src/lib/generator.ts [hunk 4]
```diff
@@ -212,17 +213,48 @@ function formatConversions(build: BuildSettings): string[] {
...
+      ['Transmission', build.drivetrain.transmission],
+      ['Driveline', build.drivetrain.driveline],
+      ['Differential', build.drivetrain.differential],
+    ]),
+    ...formatTable(['Tires And Rims', ''], [
+      ['Compound', build.tiresAndRims.compound],
+      ['Width', `Front ${build.tiresAndRims.width.front} mm, Rear ${build.tiresAndRims.width.rear} mm`],
+      ['Rim Style', `${build.tiresAndRims.rimStyle.type} ${build.tiresAndRims.rimStyle.name}`],
+      ['Rim Size', `Front ${build.tiresAndRims.rimSize.front} in, Rear ${build.tiresAndRims.rimSize.rear} in`],
+    ]),
```
Comment (SharpSeeEr) [inline src/lib/generator.ts:255]: Also moved from a separate function.  In this case Track Width was removed as well.  Reasoning?

File: src/lib/useFormattingForm.ts [hunk 5]
```diff
@@ -94,7 +94,7 @@ function createFormattingForm(): SettingsForm {
     build: {
       conversions: {
         engine: 'Stock',
-        drivetrain: DriveType.na,
+        drivetrain: DriveType.awd,
```
Comment (SharpSeeEr) [inline src/lib/useFormattingForm.ts:97]: By default I think this should be 'Stock'.

**Review by SharpSeeEr — COMMENTED** [review]
For now the `.idea` folder should be added to `.gitignore` and deleted from the repository (not from your local folder though)

**Comment by Ldalvik** [comment]
Looks like my IDE has a mind of its own because i only made one of those changes. Very weird! I’m gonna rollback

**Comment by SharpSeeEr** [comment]
Cancelling PR

**Pull request closed without merging**